core/src/test/scala-3/cats/derived/MonoidKSuite.scala

```diff
@@ -0,0 +1,64 @@
+package cats.derived
+
+import alleycats.*
+import cats.*
+import cats.derived.*
+import cats.laws.discipline.{MonoidKTests, SerializableTests}
+import org.scalacheck.Arbitrary
+
+class MonoidKSuite extends KittensSuite {
+  import MonoidKSuite.*
+  import TestDefns.*
+
+  def testMonoidK(context: String)(implicit
+      complexProduct: MonoidK[ComplexProduct],
+      caseClassWOption: MonoidK[CaseClassWOption],
+      boxMul: MonoidK[BoxMul]
+  ): Unit = {
```

Comment:
WDYT of turning this into inline function to remove the implicit parameters like similar tests in other files?

Reply:
Oh sorry, I missed that we'd done that in the other scala 3 suites. Good call 👍